core/src/components/avatar/test/states/avatar.e2e.ts

```diff
@@ -0,0 +1,27 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+/**
+ * Avatar does not test RTL behaviors.
+ * Usages of Avatar in slots are tested in components that use Avatar.
+ */
+configs({ directions: ['ltr'], modes: ['ionic-md'] }).forEach(({ config, screenshot, title }) => {
+  test.describe(title('avatar: states'), () => {
+    test('should not have visual regressions', async ({ page }) => {
+      await page.setContent(
+        ` 
+        <div id="container"> 
+          <ion-avatar id="avatar-characters-disabled" disabled> AV </ion-avatar>
+          <ion-avatar id="avatar-characters-disabled" disabled>
```

Comment:
```suggestion
          <ion-avatar disabled> AV </ion-avatar>
          <ion-avatar disabled>
```

You can remove the ids here too but shouldn't need to generate screenshots again after